src/main/resources/data/createaddition/tags/items/plant_foods.json

```diff
@@ -7,6 +7,7 @@
     "minecraft:sweet_berries",
     "minecraft:beetroot",
     "minecraft:glow_berries",
-    "minecraft:carrot"
+    "minecraft:carrot",
+    "minecraft:wheat"
```

Comment:
revert this change and instead add this recipe: https://github.com/mrh0/createaddition/blob/1.19.2/src/main/resources/data/createaddition/recipes/mixing/biomass_from_crops.json